src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -223,6 +223,17 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     // TODO
   }
 
+  override async fetchTokenInfo(): Promise<IToken> {
+    // TODO if needed
+    // @ts-expect-error
+    return undefined;
+  }
+
+  // @ts-expect-error
```

Comment:
I think the base adapter should be written in a way that would not force us to use such hacks.